File: FITCC.UI/admin_fitcc/lib/models/rezultat.dart
import 'package:admin_fitcc/models/projekat.dart';
import 'package:json_annotation/json_annotation.dart';

part 'rezultat.g.dart';

@JsonSerializable(explicitToJson: true)
class Rezultat {
  int rezultatId;
   int bod;
   String napomena;
   int projekatId;
   Projekat projekat; 
  Rezultat(
 this.rezultatId,
    this.bod,
    this.napomena,
    this.projekatId,
    this.projekat
    );

  factory Rezultat.fromJson(Map<String, dynamic> json) => _$RezultatFromJson(json);
  Map<String, dynamic> toJson() => _$RezultatToJson(this);
}

File: FITCCRS2App.Models/Models/Rezultat.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace FITCCRS2App.Models.Models
{
	public class Rezultat
	{
        public int RezultatId { get; set; }
        public string Napomena { get; set; }
        public int Bod { get; set; }

        public int ProjekatId { get; set; }
        public virtual Projekat Projekat { get; set; } = null!;

    }
}

